tools/harness-thci/ARM.py

```diff
@@ -1556,25 +1556,49 @@ def startNativeCommissioner(self, strPSKc='GRLpassWord'):
         pass
 
     def startCollapsedCommissioner(self):
-        pass
+        """start openthread stack"""
```

Comment:
Ok, sure. I will refine the code before it is ready for merge, work in progress now.